.github/workflows/push.yml

```diff
@@ -80,6 +80,10 @@ jobs:
       - name: Run tests
         run: make test
 
+      - name: Verify -inprocess feature of acceptance test by running one test
```

Comment:
We can skip this in CI? The `-inprocess` feature is meant to be only used by us during dev.

Reply:
This a check that it keeps working.

Reply:
This was removed.